scipy/fft/tests/test_basic.py

```diff
@@ -333,8 +333,11 @@ def test_dtypes_real(self, dtype, xp):
 
     @pytest.mark.parametrize("dtype", ["complex64", "complex128"])
     def test_dtypes_complex(self, dtype, xp):
+        # Trick to get the array-api-compat namespace for dask
+        # (otherwise the "naked" dask.array asarray does not respect
+        # the input dtype)
```

Comment:
I think I fixed this in https://github.com/dask/dask/pull/11288?

Reply:
I skimmed that patch really quick - but at a quick glance it looks like that only fixes the issue when the input is a dask array.

When the input is a numpy array it looks like we go into the ``from_array`` codepath which is not passed the dtype option?

Reply:
If my patch didn't fix it then this should be a new bug report for Dask - we shouldn't have to work around that.

Reply:
I'll poke around more inside dask to double check if this is still an issue.

Reply:
Seems to be fixed in 2024.12.1 at least - gonna take this out.
Maybe I was on too old of a dask version.

Reply:
Fixed in https://github.com/dask/dask/pull/11586